### PR TITLE
Fetch students, save them in redux

### PR DESCRIPTION
**Pull request by Kikisek**
Created actions and a reducer to fetch all students.
Sorted students alphabetically according to the last word in their name.
Saved students in redux.

Removed unnecessary placeholder checklist data.

### CODE REVIEW

File: src/app/store/actions/getStudentsData.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+});
+
+const getStudentsData = () => dispatch => {
+  fetch(`${BASE_URL}/students`)
+    .then(response => response.json())
+    .then(json => {
+      const students = json;
+
+      // sort alphabetically by last name
+      students.sort((a, b) => {
```
Comment (susiekims) [inline src/app/store/actions/getStudentsData.js:20]: cool!

File: src/app/components/Students.js [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+    getStudentsData();
+  }
 
-export default Students;
+  render() {
+    return <h1>Students</h1>;
+  }
+}
+
+const mapStateToProps = state => ({ students: state.students.data });
```
Comment (susiekims) [inline src/app/components/Students.js]: i think it should be:
```suggestion
const mapStateToProps = state => ({ students: state.students.studentsData });
```
:)

**Review by susiekims — CHANGES_REQUESTED** [review]
Looks like the data is not getting written to the component's props, i think you can either change `data` to `studentsData` in `Students` or vice versa in the reducer. maybe the second option is better so it's consistent with the company reducer? otherwise great work as usual :)

File: src/app/components/Students.js (hunk 2 above)
Comment (Kikisek) [reply src/app/components/Students.js]: Awesome catch! Thanks a lot 👍

**Review by susiekims — APPROVED** [review]
👍 👍 👍

**Pull request merged**